### PR TITLE
Adding more synthetic data

### PR DESCRIPTION
**Pull request by srbhchandra**
Changes in PR:
 - Using the IAM synthetic paragraphs validation dataset (along with the IAM paragraphs validation dataset) for validation
 - Generating a lot more synthetic paragraphs in `generate_synthetic_paragraphs()`

Wandb run - https://wandb.ai/fullstackdeeplearning/fsdl-text-recognizer-2022-training/runs/1w6b7fc9 
The run was unsuccessful (`validation loss` kept increasing after a few epochs) because of 1 bug: we didn't set `IAMSyntheticParagraphs.transform.scale_factor = 1` which was needed since we are using the IAM synthetic paragraphs validation dataset. So `IAMSyntheticParagraphs.transform.scale_factor` defaulted to 2.

Signed-off-by: srbhchandra <srbhchandra@gmail.com>

### CODE REVIEW

**Comment by srbhchandra** [comment]
Closing this PR since it did not lead to a specific successful run and the main change of this PR is absorbed into https://github.com/full-stack-deep-learning/fsdl-text-recognizer-2022/pull/21.
